tests/unit/onyxClearNativeStorageTest.js

```diff
@@ -0,0 +1,165 @@
+import AsyncStorageMock from '@react-native-async-storage/async-storage';
```

Comment:
Ah, that's interesting and quite magical. 🤔 

I would probably favor the second import since it makes it more clear. If we stick with the first import, I think just adding a comment to explain that it's magically loading the mock and not the real thing would help.

Reply:
Importing it directly is a good idea.